spec/dummy/db/schema.rb

```diff
@@ -10,14 +10,11 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20130502195504) do
-
-  # These are extensions that must be enabled in order to support this database
-  enable_extension "plpgsql"
+ActiveRecord::Schema.define(version: 2018_06_05_142534) do
```

Comment:
Not entirely sure why some of these changes are here -- possibly this was originally created with a different database.yml config. The tests are passing though.